src/Widgets/EligibilityModal/DesktopModal/index.tsx

```diff
@@ -1,16 +1,19 @@
 import cx from 'classnames'
 import React, { FC } from 'react'
+import {Card} from 'types'
```

Comment:
```suggestion
import { Card } from 'types'
```

Strange Prettier did not catch this